O3R/Docker/README.md

```diff
@@ -0,0 +1,10 @@
+# README - Docker on O3R
+
+| Table of content|
+|-|
+| [Getting started with Docker](../Docker/docker.md)|
+| [Deployment on board](../Docker/deployVPU.md)|
+| [Autostarting the container](../Docker/autostart.md)|
+| [Enabling GPU usage](../Docker/gpu.md)|
+
+This section explains the handling and deployment of custom specific containers for the O3R.
```

Comment:
custom containers or customer specific containers? I think "custom specific containers" is a confusing prasing